### DISCUSSION TITLE
Tailwind stopped worked when files got moved to /src

### DISCUSSION THREAD

**Discussion by guilhermeadf** [tailwindlabs/tailwindcss#12018] [Help] [answered]
After moving my files to the newly created /src folder and updating the `tailwind.config.js` file accordingly, I can't seem to make tailwind work as it used to before the change. Any ideas?

<img width="824" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/36577631/8a49bdde-321f-43a0-b8b8-c9b3fb926e24">

**Comment by wongjn** [comment, marked as answer]
Consider trying:
- Restarting the process that runs Tailwind.
- Ensure `.js`, `.jsx`, `.ts` and/or `.tsx` files in the `src` folder contain Tailwind classes.
- [String concatenation/interpolation is not used to build class names](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 
- The CSS file that contains the `@tailwind` directives is processed by Tailwind and included in the resulting webpage. 

Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.
